### PR TITLE
Operator upgrade steps improvement

### PR DESCRIPTION
**Pull request by randeepsharma**
# Description
Improve documentation for operator upgrade and driver upgrade via operator

# GitHub Issues
List the GitHub issues impacted by this PR:
https://github.com/dell/csm/issues/127

| GitHub Issue # |
https://github.com/dell/csm/issues/127
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rensyct — DISMISSED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
@randeepsharma : Please rebase the code with the branch

**Review by bpjain2004 — DISMISSED** [review]

File: content/docs/csidriver/upgradation/drivers/operator.md [hunk 1]
```diff
@@ -7,16 +7,25 @@ weight: 1
...
-```
-$ bash scripts/install.sh --upgrade
-```
+1. Clone the [Dell CSI Operator repository](https://github.com/dell/dell-csi-operator).
+2. git checkout dell-csi-operator-<your-version>
+3. Execute `bash scripts/install.sh --upgrade`  . This command will install the latest version of the operator.
+>Note: Dell CSI Operator version 1.4.0 and higher would install to the 'dell-csi-operator' namespace by default.
 
 ### Using OLM
+This is usually on OpenShift Platforms where OLM manages the operator upgrade.
```
[the remark this answers is not in the document]
Comment (shanmydell) [reply content/docs/csidriver/upgradation/drivers/operator.md:22]: @randeepsharma : Requesting your inputs here

**Comment by shanmydell** [comment]
@randeepsharma : There was a formatting issue, 
image
Is this format okay?

**Comment by randeepsharma** [comment]
> @randeepsharma : There was a formatting issue, image
> 
> Is this format okay?

Yes.

File: content/docs/csidriver/upgradation/drivers/operator.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (randeepsharma) [reply content/docs/csidriver/upgradation/drivers/operator.md]: removed this line based on discussion

**Review by shanmydell — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Pull request merged**